components/Datepicker/src/index.tsx

```diff
@@ -0,0 +1,400 @@
+import React, { InputHTMLAttributes, KeyboardEvent, RefObject } from 'react';
+import clsx from 'clsx';
+import {
+  format,
+  Locale,
+  getWeeksInMonth,
+  formatISO,
+  setDay,
+  addWeeks,
+  subMonths,
+  addMonths,
+  isSameMonth,
+  setDate,
+  isEqual,
+  addDays,
+  subDays,
+  startOfWeek,
+  endOfWeek,
+  subYears,
+  addYears,
+  subWeeks,
+  getWeekOfMonth,
+  getDay,
+  getDate,
+  setMonth,
+  getMonth,
+  setYear,
+  getYear,
+} from 'date-fns';
+import { nl } from './locale';
+
+import { CalendarIcon, AlertTriangleIcon, ChevronRightIcon, ChevronLeftIcon } from '@gemeente-denhaag/icons';
+
+import './datepicker.css';
+
+export interface DatepickerProps extends InputHTMLAttributes<HTMLInputElement> {
+  /**
+   * The ref for the outer HTML div.
+   */
+  ref?: RefObject<HTMLDivElement>;
+  /**
+   * The ref for the actual input HTML element.
+   */
+  inputRef?: RefObject<HTMLInputElement>;
+  /**
+   * Whether the input field is in a error state.
+   */
+  error?: boolean;
+  /**
+   * Whether the input field is disabled.
+   */
+  disabled?: boolean;
```

Comment:
The `disabled` prop should already be supplied by the [InputHTMLAttributes](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement) interface

Reply:
Fixed!